account-and-billing/account-and-billing.constant.ts

```diff
@@ -0,0 +1,51 @@
+export class AccountAndBillingConstant {
+    static readonly attributes = Object.freeze({
+        text: Object.freeze({
+            accountAndBilling: 'Account and Billing',
+            pastPayments: 'Past Payments',
+            accountId: 'Account ID',
+            customerSupport: 'Customer Support',
+            primaryContact: 'Primary Contact',
+            exportData: 'Export Data',
+            downloadLink: 'Download FogBugz data now',
+            closeAccount: 'Close Account',
+            closedAccountsCannotBeRecovered: 'Closed accounts cannot be recovered',
+            editProducts: 'Edit Products',
+            paymentPlans: 'Payment Plans',
+            addCreditCard: 'Add Credit Card',
+            creditCard: 'Credit Card',
+            nextBill: 'Next Bill',
+            contactCustomerSupport: 'Contact Customer Support',
+            supportUrl: 'https://support.fogbugz.com/hc/en-us',
+            blank: '_blank',
+
+        }),
+        class: Object.freeze({
+            helpText: 'help-text',
+            editProducts: 'edit-products',
+            upgradeProduct: 'upgrade-product',
+            paymentPlans: 'payment-plans',
+            addCreditCard: 'update_plan',
+            creditCard: 'dlg',
+            nextBill: 'next-bill',
+            closeAccount: 'close-account',
+            whenBill: 'when-bill',
+            product: 'product',
+            productPaid: 'product-paid',
+        }),
+        ids: Object.freeze({
+            accountForm: 'account_form',
+            editProducts: 'editable',
+            cancelButton: 'Button_Cancel',
+            editEditProductFeatures: 'editEditProductFeatures',
+        }),
+        names: Object.freeze({
+            authorize: 'authorize',
+            fogBugz: 'FogBugz',
+        }),
+        href: Object.freeze({
+            support: 'https://support.fogbugz.com/',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400120)
```suggestion
            support: "https://support.fogbugz.com/",
```